hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/sink/clustering/HoodieFlinkClusteringJob.java

```diff
@@ -259,15 +287,18 @@ private void cluster() throws Exception {
       if (!clusteringPlanOption.isPresent()) {
         // do nothing.
         LOG.info("No clustering plan scheduled, turns on the clustering plan schedule with --schedule option");
+        executeDummyPipeline();
         return;
       }
 
+      clusteringInstant = clusteringPlanOption.get().getLeft();
       HoodieClusteringPlan clusteringPlan = clusteringPlanOption.get().getRight();
```

Comment:
Is there any special reason we need to overwrite the `clusteringInstant` from the `clusteringPlanOption` ?

Reply:
No special reasons, I will remove it.